eventually/customprofile/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
```

Comment:
comment this line "from django.shortcuts import render" It would remove some mistakes in travis